index.html

```diff
@@ -83,9 +84,219 @@ <h1 class="headline_title">
+        </div>
+
+        <div class="project_card">
+          <div class="img_holder"></div>
+          <h2 class="large_title">
+            Multi-Post Stories <br />
+            Gain+Glory
+          </h2>
+          <ul class="tools_flex">
+            <li class="tool_item">
+              <button class="tools_design">Ruby on rails</button>
+            </li>
+            <li>
+              <button class="tools_design">css</button>
+            </li>
+            <li>
+              <button class="tools_design">JavScript</button>
+            </li>
+            <li>
+              <button class="tools_design">html</button>
+            </li>
+          </ul>
+          <button class="btngreen">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <section id="about-me" class="container">
+      <h1>About me</h1>
+      <div class="img3"></div>
+      <p>
+          I can help you build a product , feature or website Look through some of my work and experience! If you like
+          what you see and have a project you need coded,
+          don't hestiate to contact me.
+      </p>
+      <div class="img4"></div>
+      <button class="btngreen">
+          Get My Resume
+      </button>
+  </section>
+
+    <section>
+      <div class="card">
+          <img class="logo-img" src="assets/ic_languages.svg" alt="Languages">
+          <h2>
+              Languages
+          </h2>
+          <ul class="tools_flex">
+              <li><button class="skills_design">JavaScript</button></li>
+              <li><button class="skills_design">Ruby</button></li>
+              <li><button class="skills_design">Html</button></li>
+              <li><button class="skills_design">CSS</button></li>
+          </ul>
+      </div>
+
+      <div class="card">
+          <img class="logo-img" src="assets/ic_frameworks.svg" alt="Frameworks">
+          <h2>
+              Frameworks
+          </h2>
+          <ul class="tools_flex">
+              <li><button class="skills_design">React.js</button></li>
+              <li><button class="skills_design">Ruby on rails</button></li>
+              <li><button class="skills_design">RSpec</button></li>
+              <li><button class="skills_design">Capybara</button></li>
+              <li><button class="skills_design">Selenium</button></li>
+          </ul>
+      </div>
+
+      <div class="card">
+          <img class="logo-img" src="assets/ic_skills.svg" alt="Skills">
+          <h2>
+              Skills
+          </h2>
+          <ul class="tools_flex">
+              <li><button class="skills_design">Database Management</button></li>
+              <li><button class="skills_design">Version Control</button></li>
+              <li><button class="skills_design">CLI</button></li>
+              <li><button class="skills_design">Web Development</button></li>
+              <li><button class="skills_design">API Design</button></li>
+          </ul>
+      </div>
```

Comment:
Since you are trying to replicate the figma design and building a responsive website, the mobile version design should like like the actual design in terms of positioning, margins, paddings etc for all screen widths of the mobile version ( 375px to 768px). However, at certain screen widths, the margin and paddings changes, the position also is different as shown in the image. Kindly check and update your design to match the actual one and ensure that at all screen widths the mobile version looks the same. 

### Your design 

![Screenshot from 2023-05-05 12-45-41](https://user-images.githubusercontent.com/65233860/236461979-6e98a05c-5fda-4ebf-a4da-fedd01f6eccf.png)

### Figma design 

![Screenshot from 2023-05-05 12-44-47](https://user-images.githubusercontent.com/65233860/236462022-751267d5-ba04-4fd3-bba8-708c3c36fd64.png)